README.md

```diff
@@ -26,19 +35,24 @@ Chat with the Cockatrice developers on Gitter. Come here to talk about the appli
 
 
 # Community Resources
+- [Cockatrice Official Site](https://cockatrice.github.io)
 - [Cockatrice Official Wiki](https://github.com/Cockatrice/Cockatrice/wiki)
-- [reddit r/Cockatrice](http://reddit.com/r/cockatrice)
-- [Woogerworks](http://www.woogerworks.com) / [Chickatrice] (http://www.chickatrice.net/) / [Poixen](http://www.poixen.com/) (incomplete Serverlist)<br>
+- [reddit r/Cockatrice](https://reddit.com/r/cockatrice)
 
 
 # Translation Status [![Cockatrice on Transiflex](https://ds0k0en9abmn1.cloudfront.net/static/charts/images/tx-logo-micro.646b0065fce6.png)](https://www.transifex.com/projects/p/cockatrice/)
 
 Cockatrice uses Transifex for translations. You can help us bring Cockatrice/Oracle to your language or edit single wordings by clicking on the associated charts below.<br>
-Our [project page](https://www.transifex.com/projects/p/cockatrice/) offers a detailed overview for contributors.
-
-Language statistics for `Cockatrice` *(on the left)* and `Oracle` *(on the right)*:
+Our [Transifex project page](https://www.transifex.com/projects/p/cockatrice/) offers a detailed overview of all language statistics.
 
-[![Cockatrice translations](https://www.transifex.com/projects/p/cockatrice/resource/cockatrice/chart/image_png)](https://www.transifex.com/projects/p/cockatrice/resource/cockatrice/)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[![Oracle translations](https://www.transifex.com/projects/p/cockatrice/resource/oracle/chart/image_png)](https://www.transifex.com/projects/p/cockatrice/resource/oracle/)
+<!-- spaces (&nbsp;) needed for propper layouting, no character entity for tab available and <pre> not helping -->
```

Comment:
Gross. Does this get the layout you want on multiple browsers / window sizes? I'm more inclined to make the layout work without resorting to hacks like this.

That said, if it looks good normally and at least OK everywhere, go on